app/javascript/controllers/transcript_controller.js

```diff
@@ -88,19 +88,12 @@ export default class extends Controller {
   }
 
   edit(event) {
-    this.editTarget.innerHTML = `<strong class="btn-icon" data-controller="tooltip" data-bs-toggle="tooltip" data-bs-position="bottom" title="Save"><i class="fa-solid fa-floppy-disk"></i></strong>`
+    this.editTarget.innerHTML = `<strong class="btn-icon text-primary" data-controller="tooltip" data-bs-toggle="tooltip" data-bs-position="bottom" title="Save"><i class="fa-solid fa-floppy-disk"></i></strong>`
     this.editTarget.dataset.action = "click->transcript#save"
     this.contentTarget.contentEditable = "true";
+    this.contentTarget.focus();
     const bubble = document.createElement('div');
     bubble.classList.add('floating-bubble');
-    bubble.innerHTML = `
-      <button onclick="document.execCommand('bold', false, null)"><i class="fa-solid fa-bold"></i></button>
-      <button onclick="document.execCommand('italic', false, null)"><i class="fa-solid fa-italic"></i></button>
-      <button onclick="document.execCommand('underline', false, null)"><i class="fa-solid fa-underline"></i></button>
-      <button id="highlight-button"><i class="fa-solid fa-highlighter"></i></button>
-      <button id="remove-format-button"><i class="fa-solid fa-eraser"></i></button>
-
-    `;
```

Comment:
now we build the HTML for the bubble every time the selection changes, so we can set the button state according to the current state of the selection (see below)